hplsql/src/main/java/org/apache/hive/hplsql/Var.java

```diff
@@ -601,7 +602,7 @@ else if (type == Type.STRING) {
       return (String)value;
     }
     else if (type == Type.DATE) {
-      return String.format("DATE '%s'", value);
+      return ((Date)value).toString();
```

Comment:
Is it safe to remove the `DATE` keyword? Let's say we have `2024-04-09`. Is it a string or a date literal?

Reply:
As explained in https://github.com/apache/hive/pull/5182#discussion_r1570337451 comment, this was added by me only just reverted back the change done as a part of [HIVE-27492](https://issues.apache.org/jira/browse/HIVE-27492) fix.